cmd/tools/testwick/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/mattermost/mattermost-cloud/cmd/tools/testwick
```

Comment:
I wonder what are advantages of adding it as a separate module?

Reply:
avoid bumping aws sdk version etc in the main go module